deploy/deployChangedPackages.mjs

```diff
@@ -40,28 +44,51 @@ const go = async () => {
     const newTSConfig = fs.readFileSync(localPackageJSONPath, "utf-8");
     const pkgJSON = JSON.parse(newTSConfig);
 
+    // We'll need to map back from the filename in the npm package to the
+    // generated file in baselines inside the git tag
+    const thisPackageMeta = packages.find((p) => p.name === pkgJSON.name);
+
     const dtsFiles = fs
       .readdirSync(join(generatedDir, dirName))
       .filter((f) => f.endsWith(".d.ts"));
 
+    /** @type {string[]} */
+    let releaseNotes = [];
+
     // Look through each .d.ts file included in a package to
     // determine if anything has changed
     let upload = false;
     for (const file of dtsFiles) {
+      const originalFilename = basename(
+        thisPackageMeta.files.find((f) => f.to === file).from
+      );
+
       const generatedDTSPath = join(generatedDir, dirName, file);
       const generatedDTSContent = fs.readFileSync(generatedDTSPath, "utf8");
-      const unpkgURL = `https://unpkg.com/${pkgJSON.name}/${file}`;
+
+      // This assumes we'll only _ever_ ship patches, which may change in the
+      // future someday.
+      const [maj, min, patch] = pkgJSON.version.split(".");
+      const olderVersion = `${maj}.${min}.${patch - 1}`;
```

Comment:
(Just to note for future reference: `git describe --tags --match "@types/web@*" --abbrev=0` can do the same thing)